Integrations/integration-Verodin.yml

```
@@ -0,0 +1,859 @@
+            });
+            if (!res.ID && id) {
+                res.ID = id;
+            }
+            return res;
+        };
+        if (data instanceof Array) {
+            var res = [];
+            for (var j=0; j < data.length; j++) {
+                res.push(createContextSingle(data[j]));
+            }
+            return res;
+        }
+        return createContextSingle(data);
+    }
+
+    function mapObjFunction(mapFields, filter) {
+        var transformSingleObj= function(obj) {
+            var res = {};
+            mapFields.forEach(function(f) {
+                res[f.to] = dq(obj, f.from);
+            });
+            if (filter && !filter(res)) {
+                return undefined;
+            }
+            return res;
+        };
+        return function(obj) {
+            if (obj instanceof Array) {
+                var res = [];
+                for (var j=0; j < obj.length; j++) {
+                    var current = transformSingleObj(obj[j]);
+                    if (current) {
+                        res.push(current);
+                    }
+                }
+                return res;
+            }
+            return transformSingleObj(obj);
+        };
+    }
+
+    var sendRequest = function(url, method, args) {
+        var headers = {};
+        var httpParams = {
+           Method: method,
+           Username: params.credentials.identifier,
+           Password: params.credentials.password,
+        };
+        var res = http(
+                baseUrl + url + (method === 'GET' && method ? encodeToURLQuery(args) : ''),
+                httpParams,
+                params.insecure,
+                params.proxy,
+                false
+            );
+        if (res.StatusCode < 200 || res.StatusCode >= 300) {
+            throw 'Request ' + url + ' failed, request status code: ' + res.StatusCode + ' and Body: ' + res.Body + '.';
+        }
+        if (res.StatusCode === 204) {
```

What is this ?
Dont you want to return a proper error response ?